picard/coverart/__init__.py

```diff
@@ -130,7 +137,7 @@ def download_next_in_queue(self):
                     p = provider(self)
                     if p.enabled():
                         log.debug("Trying cover art provider %s ..." % name)
-                        ret = p.queue_downloads()
+                        ret = p.queue_images()
```

Comment:
The only one I know of: https://github.com/musicbrainz/picard-plugins/tree/master/plugins/fanarttv